src/FocusCore/Responses/User/EditUserResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace FocusCore.Responses.User
+{
+    public class EditUserResponse
```

Comment:
Since this is only used by that MediatR feature and isn't used by both FocusApp and FocusAPI, could we move this to be a nested class inside of this MediatR feature?